libs/category/src/models/requests/filter-action.ts

```diff
@@ -1,19 +1,21 @@
-export enum DaffCategoryFilterActionEnum {
-	Equal = 'eq',
-	FromTo = 'fromto',
-	In = 'in',
-	Match = 'match'
-}
+import { DaffCategoryFilterType } from '../category-filter';
 
 /**
  * The separator between a range type filter's values
  * e.g. "price": "30-40"
  */
 export const DaffCategoryFromToFilterSeparator = '-';
 
-export interface DaffCategoryFilterAction {
-	action: DaffCategoryFilterActionEnum;
+export interface DaffToggledCategoryFilterAction {
+	type: DaffCategoryFilterType;
 	name: string;
 	value: string;
 	label?: string;
 }
+
+export interface DaffCategoryFilterAction {
```

Comment:
The trailing "Action" here is easily confused with a Redux action, let's suffix with `Request` instead.